tqdm/tests/tests_tqdm.py

```diff
@@ -9,10 +9,14 @@
 import os
 from nose import with_setup
 from nose.plugins.skip import SkipTest
+from nose.tools import assert_raises
 
 from tqdm import tqdm
 from tqdm import trange
 from tqdm import TqdmDeprecationWarning
+from tqdm._tqdm import TMonitor
+
+from time import sleep
```

Comment:
No we (implicitly) adopted the following style advised in Python doc:

```
from __future__ import x

# tqdm internal import
import ._core as tqdm
from ._utils import x

# Third party modules
import x
from x import y  # from are after

# tqdm external imports (from an external module or unit test)
import tqdm
from tqdm import x
```

Ordered alphabetically and with no grouping of modules on one line (no `import os, time, etc.`) but we can group functions/class imports (`from time import sleep, time`).

But that was adopted implicitly so there may be some discrepancies here and there.